x/evm/types/utils.go

```diff
@@ -181,3 +182,11 @@ func BinSearch(lo, hi uint64, executable func(uint64) (bool, *MsgEthereumTxRespo
 func EffectiveGasPrice(baseFee *big.Int, feeCap *big.Int, tipCap *big.Int) *big.Int {
 	return math.BigMin(new(big.Int).Add(tipCap, baseFee), feeCap)
 }
+
+// HexAddress encode ethereum address without checksum, faster to run for state machine
+func HexAddress(a []byte) string {
```

Comment:
do we add any check for HexAddress

Reply:
just hex encoding, what kind of check?

Reply:
it's used by event emitting, we shouldn't deal with hex stuff in state machine in the first place.
it's to replace the checksum hex method provided by go-ethereum.